app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,45 @@
 class PostsController < ApplicationController
+  before_action :set_posts, only: [:edit, :update, :destroy]
   def index
     @posts = Post.all
   end
+
+  def new
+    @post = current_user.posts.new
+  end
+
+  def create
+    @post = current_user.posts.create(post_params)
+    if @post.save
+      redirect_to root_path
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if @post.update(post_params)
+      redirect_to root_path
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @post.destroy!
+    redirect_to root_path
+  end
```

Comment:
you are not showing notification messages here in this crud

Reply:
What notification messages ? Im not using notification messages anywhere.